tools/RunGen.h

```diff
@@ -1168,11 +1182,20 @@ class RunGen {
         config.max_iters = benchmark_max_iters;
         auto result = Halide::Tools::benchmark(benchmark_inner, config);
 
-        out() << "Benchmark for " << md->name << " produces best case of " << result.wall_time << " sec/iter (over "
-              << result.samples << " samples, "
-              << result.iterations << " iterations, "
-              << "accuracy " << std::setprecision(2) << (result.accuracy * 100.0) << "%).\n"
-              << "Best output throughput is " << (megapixels_out() / result.wall_time) << " mpix/sec.\n";
+        if (!parsable_output) {
+            out() << "Benchmark for " << md->name << " produces best case of " << result.wall_time << " sec/iter (over "
+                  << result.samples << " samples, "
+                  << result.iterations << " iterations, "
+                  << "accuracy " << std::setprecision(2) << (result.accuracy * 100.0) << "%).\n"
+                  << "Best output throughput is " << (megapixels_out() / result.wall_time) << " mpix/sec.\n";
+        } else {
+            out() << md->name << "  BEST_TIME_MSEC_PER_ITER  " << result.wall_time * 1000.f << "\n"
+                  << md->name << "  SAMPLES                  " << result.samples << "\n"
+                  << md->name << "  ITERATIONS               " << result.iterations << "\n"
+                  << md->name << "  TIMING_ACCURACY          " << result.accuracy << "\n"
+                  << md->name << "  THROUGHPUT_MPIX_PER_SEC  " << (megapixels_out() / result.wall_time) << "\n"
+                  << md->name << "  HALIDE_TARGET            " << md->target << "\n";
```

Comment:
Not for this PR, but I was thinking about distinguishing between performance of different generator param settings (mostly auto_schedule vs not), and it made me wonder if we should be embedding generator param values in the metadata. Or do we already?

Reply:
We don't, but that's a good idea. Please open an issue on it :-)